Non-void functions need a return, even if it is never reached.

--- a/lib/vf/vq.c	2018-01-04 04:38:06.000000000 +0100
+++ b/lib/vf/vq.c	2018-01-14 17:21:38.048770122 +0100
@@ -76,6 +76,7 @@
 			return 0;
 		}
 	}
+	return 0;
 }
 caryll_OrdEqFns(vq_Segment, vqsCompare);
 static void showVQS(const vq_Segment x) {
@@ -131,6 +132,7 @@
 		case VQ_DELTA:
 			return 0 == vq_compareRegion(a.val.delta.region, b.val.delta.region);
 	}
+	return false;
 }
 static void simplifyVq(MODIFY VQ *x) {
 	if (!x->shift.length) return;
